docs/introduction.md

```diff
@@ -0,0 +1,59 @@
+# Introduction to Personalization
+
+Website Personalization is the process of creating customized experiences for visitors to a website. Rather than providing a single experience for all visitors, website personalization allows companies to present visitors with unique experiences tailored to their needs and desires.
+
+## What does this SDK offer?
+
+This SDK offers both Geo and Interest targeting. Below we will go into further detail for each type of targeting.
+
+## Geotargeting Functionality
+
+Geotargeting is the method of delivering different content to visitors based on their geolocation. This includes country, region, city, and other criteria.
+
+### Default Behavior and Available Customizations
+
+Using the `get_geo` function, engineers can get location specific data from site visitors, including `country`, `region`, `city`, `continent`, `conn-speed`, `conn-type`, `lat`, `lon`, and `latlon`. Engineers can modify these data types using the `pantheon.ei.geo_allowed_values` filter.
+
+Personalized data can then be served to visitors based on any of the geo data types mentioned above.
+
+Engineers are have access to the geo data via the `pantheon.ei.before_get_geo` action. This action fires after the geo data is retrieved but before it is returned, which allows developers to hook into the geo data retrieval process and access the geo value and the type of data requested and the full passed data, if it exists.
+
+Geo data can also be retrieved using `pantheon.ei.get_geo` filter. This filter fires after the data is parsed and before it is returned making this the last stop before data is output, and allows developers to modify the requested geo data.
```

Comment:
Using a filter isn't _exactly_ the same as retrieving data. I'd tweak this a bit to be clearer/more accurate.

```suggestion
Geo data can also be retrieved and altered using `pantheon.ei.get_geo` filter. This filter fires after the data is parsed and before it is returned making this the last stop before data is output, and allows developers to modify the requested geo data.
```